frontend/test/snapshot-creators/qa-db.cy.snap.js

```diff
@@ -17,13 +17,15 @@ describe("qa databases snapshots", () => {
 
     restoreAndAuthenticate();
 
-    addMongoDatabase();
-    snapshot("mongo-4");
+    cy.wait(1000);
+    addMySQLDatabase();
+    snapshot("mysql-8");
 
     restoreAndAuthenticate();
 
-    addMySQLDatabase();
-    snapshot("mysql-8");
+    cy.wait(1000);
+    addMongoDatabase();
+    snapshot("mongo-4");
```

Comment:
Added waits here since I was running into flakiness in the DB connection step otherwise. Don't fully understand the root cause but this seems to fix it. (Also swapped the order of MySQL/Mongo which seemed to help, but might not be necessary)